app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -72,13 +72,20 @@ class BookmarkFragment : LibraryPageFragment<BookmarkNode>(), UserInteractionHan
     private var pendingBookmarkDeletionJob: (suspend () -> Unit)? = null
     private var pendingBookmarksToDelete: MutableSet<BookmarkNode> = mutableSetOf()
 
+    private var _fragmentBookmarkBinding: FragmentBookmarkBinding? = null
+    private val fragmentBookmarkBinding get() = _fragmentBookmarkBinding!!
+    private var _componentBookmarkBinding: ComponentBookmarkBinding? = null
+    private val componentBookmarkBinding get() = _componentBookmarkBinding!!
```

Comment:
Not sure you need a `ComponentBookmarkBinding`. You already have access to it by using `bookmarkView.binding`